src/v2/Apps/Show/__tests__/ShowArtworksEmptyState.jest.tsx

```diff
@@ -0,0 +1,65 @@
+import { graphql } from "react-relay"
+import { ShowArtworksEmptyStateFragmentContainer as ShowArtworksEmptyState } from "../Components/ShowArtworksEmptyState"
+import { ShowArtworksEmptyState_Test_Query } from "v2/__generated__/ShowArtworksEmptyState_Test_Query.graphql"
+import { setupTestWrapper } from "v2/Utils/setupTestWrapper"
+
+jest.unmock("react-relay")
+
+const getWrapper = setupTestWrapper<ShowArtworksEmptyState_Test_Query>({
```

Comment:
Nice 👍

Reply:
Yes! Let's be sure to socialize this nice new helper ✨